content/posts/newsletter-016/index.md

```diff
@@ -63,6 +63,25 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Game Updates
 
+### [Antorum Online]
```

Comment:
If you want this to link somewhere, you need a corresponding link at the bottom of the post (like you've done for your dev feed and Twitter handle). If you just want it to be a normal heading, you can take off the `[]` around it :)

Reply:
Shoot! I made the same mistake again. Will fix